doc/source/user_guide/tutorials/animate/index.rst

```diff
@@ -0,0 +1,28 @@
+.. _ref_tutorials_animate:
+
+=======
+Animate
+=======
+
+These tutorials demonstrate how to visualise the data in an animation.
+
+.. grid:: 1 1 3 3
+    :gutter: 2
+    :padding: 2
+    :margin: 2
+
+    .. grid-item-card:: Animate data
+       :link: ref_tutorials_animate_data
+       :link-type: ref
+       :text-align: center
+
+       This tutorial shows how to animate your results data.
+
+       +++
+       :bdg-warning:`MAPDL` :bdg-success:`LSDYNA`
```

Comment:
Is it possible to have both words in black, just the color behind changed? Also this tutorial applies to fluid results too.
I think we may have to design our own badges for the project.
https://getbootstrap.com/docs/5.0/components/badge/#sass
I only wonder how this would conflict with the `ansys-sphinx-theme`.
For now leave it as it is, we will use it as an example. Or maybe add it in a separate PR once this one is merged.